test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
+		} );
+
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
+		} );
+
+		test.afterEach( async ( { requestUtils } ) => {
+			await requestUtils.deleteAllPosts();
+		} );
+
+		test( 'default to a list of pages if there are no menus', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.insertBlock( { name: 'core/navigation' } );
+
+			// Necessary to wait for the block to be in loaded state else
+			// initial rendered state will not be resolved.
+			await page.waitForLoadState( 'networkidle' );
```

Comment:
This is normally not needed as the actions below will automatically (implicitly) wait for the result. Try removing it and see if it still works?

Reply:
It doesn't. The Page List isn't available in the List View until the network resolves. I have been trying to find a way around this.